content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -106,35 +127,24 @@ from within your Istio cluster. In this task you access
     EOF
     {{< /text >}}
 
-### Make requests to the external services
-
-1.  Exec into the pod being used as the test source. For example,
-    if you are using the `sleep` service, run the following commands:
+1.  Exec into the `sleep service` source pod:
 
     {{< text bash >}}
-    $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
     $ kubectl exec -it $SOURCE_POD -c sleep bash
     {{< /text >}}
 
-1.  Make a request to the external HTTP service:
-
-    {{< text bash >}}
-    $ curl http://httpbin.org/headers
-    {{< /text >}}
-
 1.  Make a request to the external HTTPS service:
 
     {{< text bash >}}
-    $ curl https://www.google.com
+    # curl https://www.google.com
```

Comment:
I think `$` is required by the site markup rules.

Reply:
That's a shame